src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newObject = {};
```

Comment:
it is better to name variables with meaningful names
```suggestion
  const newRobot = {};
```